addons/account/static/src/xml/account_payment.xml

```diff
@@ -6,18 +6,22 @@
         <div>
             <t t-if="outstanding">
                 <div>
-                    <strong class="float-left" id="outstanding"><t t-esc="title"></t></strong>
+                    <strong class="float-left" id="outstanding_title"><t t-esc="title"></t></strong>
                 </div>
             </t>
             <table style="width:100%;">
                 <t t-foreach="lines" t-as="line">
                     <tr>
                     <t t-if="outstanding">
                         <td>
-                            <a title="assign to invoice" role="button" class="oe_form_field btn btn-link outstanding_credit_assign" t-att-data-id="line.id" style="margin-right: 10px;" href="#" data-toggle="tooltip">Add</a>
+                            <a title="assign to invoice" role="button" class="oe_form_field btn btn-secondary outstanding_credit_assign"
+                               t-att-data-id="line.id" style="margin-right: 0px; padding-left: 5px; padding-right: 5px;"
+                               href="#" data-toggle="tooltip">Add</a>
                         </td>
-                        <td style="max-width: 30em;">
-                            <div class="oe_form_field" style="margin-right: 30px; text-overflow: ellipsis; overflow: hidden; white-space: nowrap;" t-att-title="line.date" data-toggle="tooltip"><t t-esc="line.journal_name"></t></div>
+                        <td style="max-width: 11em;">
+                            <a t-att-title="line.journal_name" role="button" class="oe_form_field btn btn-link outstanding_credit_view_journal"
+                               style="margin-right: 5px; text-overflow: ellipsis; overflow: hidden; white-space: nowrap; padding-left: 0px; width:100%; text-align:left;"
+                               t-att-data-id="line.move_id" data-toggle="tooltip"><t t-esc="line.journal_name"></t></a>
```

Comment:
same here
Also, `<t t-esc="line.journal_name"></t>` can be cleaner by adding `t-esc="line.journal_name"` directly inside the `<a/>`